src/Vegetation/PlantHydraulics.jl

```diff
@@ -418,13 +464,17 @@ function make_compute_exp_tendency(model::PlantHydraulicsModel, _)
         n_leaf = model.n_leaf
         fa = p.canopy.hydraulics.fa
         fa_roots = p.canopy.hydraulics.fa_roots
-
+        FT = eltype(t)
         @inbounds for i in 1:(n_stem + n_leaf)
-            AIdz =
+            # if the area_index is zero, fa is zero.
```

Comment:
Is `fa` only for RAI or for all AIs?

Reply:
it is for all fluxes - so the root flux is multiplied by the mean of SAI and RAI, and the flux between the stem and leaves are multiplide by the mean of SAI and LAI, and then transpiration is proportional to LAI as well.

Reply:
in the terminology of finite difference, the area indicies and fluxes live on faces, while theta is on cell centers..